code/headers/st7735_buffered.hpp

```diff
@@ -52,16 +82,35 @@ namespace r2d2::display {
          * @param y
          * @param width
          * @param height
-         * @param data
+         * @param data Data is the color of all pixels
          */
         void set_pixels(uint16_t x, uint16_t y, uint16_t width, uint16_t height,
-                        const uint16_t data) override;
+                        const uint16_t data) override {
+
+            uint16_t g = __REV16(data); // make a copy and reverse byte order
+
+            for (std::size_t v = 0; v < height; v++) {
```

Comment:
The variable names v and t make it really hard to reason about what is actually happening in the body of the loop. Could you refactor this?

Reply:
should allready be fixed in the latest version